includes/query_processor.inc

```diff
@@ -207,17 +207,21 @@ class IslandoraSolrQueryProcessor {
         $start = $value['solr_field_settings']['range_facet_start'];
         $end = $value['solr_field_settings']['range_facet_end'];
         $gap = $value['solr_field_settings']['range_facet_gap'];
-        // Add date facet.
+        // Add date and range facet to support both old and new facets.
         $params_date_facets["facet.date"][] = $field;
+        $params_date_facets["facet.range"][] = $field;
```

Comment:
Calculation of both where both are supported seems like an unnecessary strain on Solr.